documentation/modules/con_migration-of-luks-encrypted-disks.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// assembly_planning-migration-vmware.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="con_migration-of-luks-encrypted-disks_{context}"]
+
+= Migration of LUKS-encrypted disks
+
+[role="_abstract"]
+You can migrate virtual machines (VMs) with Linux Unified Key Setup (LUKS)-encrypted disks from VMware vSphere to Red Hat OpenShift Virtualization by enabling Network-Bound Disk Encryption (NBDE) with Clevis. Alternatively, you can manually add passphrases for LUKS-encrypted devices in your migration plan. 
+
+Components of NBDE::
+
+* *Migration Toolkit for Virtualization (MTV):* Transfers the data of VMs with LUKS-encrypted disks from the source environment to the target OpenShift Virtualization cluster. The data transfer is based on MTV's raw copy mode, which copies the encrypted data bit-for-bit, without modifying the underlying encryption.
```

Comment:
`The data transfer is based on MTV's raw copy mode, which copies the encrypted data bit-for-bit, without modifying the underlying encryption.`

This could be missunderstood as why need the passphrases for LUKS-encrypted disks.
What we do is when the virt-v2v is doing the guest conversion we read only nessary blocks, we dont' read whole disk. For LUKS it decrypts those required blocks and reencrypts those blocks locally with the same key. This way it the source will not get affected and the VM can read the blocks with the same key.
So the data are getting changed, the raw copy mode was as workaround to the NBDE.